types/manager-types.yaml

```diff
@@ -291,6 +303,14 @@ node_types:
               default: components/rabbitmq/scripts/stop.py
             fabric_env: *simple_fabric_env
 
+      cloudify.interfaces.sanitycheck:
+        postinstall:
+          implementation: fabric.fabric_plugin.tasks.run_script
+          inputs:
+            script_path:
+              default: components/checks/rabbitmq_check.py
```

Comment:
oh whoops, sorry, i broke a merge and committed this stuff that i used for debugging some time ago :X